packages/components/src/Typeahead/Typeahead.component.renderers.js

```diff
@@ -119,6 +120,16 @@ export function renderItemsContainerFactory(
 		children: PropTypes.node,
 	};
 
+	if (searching && renders.renderSearching) {
+		return renders.renderSearching;
+	} else if (noResult && loading && renders.renderLoading) {
+		return renders.renderLoading;
+	} else if (noResult && renders.renderNoResult) {
+		return renders.renderNoResult;
+	} else if (!noResult && !searching && !loading && renders.renderResults) {
+		return renders.renderResults;
+	}
+
```

Comment:
Note when you want to customize you have two options: customize all the container (drop all existing markup) or just let the control over the content.
I just want to be sure you want that.
The other options is to replace `content` in ItemsContainerComponent.

Reply:
@jmfrancois you're right. it can be achieved by passing content to Typeahead.
thanks for pointing this out. I will close this one.